compiler/rustc_target/src/spec/mod.rs

```diff
@@ -504,6 +504,36 @@ impl SplitDebuginfo {
             SplitDebuginfo::Unpacked => "unpacked",
         }
     }
+
+    /// Returns `true` if the split debuginfo configuration indicated by `self` is stable on
+    /// `target`.
+    pub fn is_stable_on_platform(&self, target: &TargetOptions) -> bool {
```

Comment:
I don't see this function being used anywhere. Did you mean to use it in `compiler/rustc_session/src/config.rs`?

Reply:
I've removed this function in the latest changes.